src/main/java/io/confluent/connect/jdbc/JdbcSourceConnectorConfig.java

```diff
@@ -16,30 +16,40 @@
 
 package io.confluent.connect.jdbc;
 
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigDef.Importance;
+import org.apache.kafka.common.config.ConfigDef.Recommender;
+import org.apache.kafka.common.config.ConfigDef.Type;
+import org.apache.kafka.common.config.ConfigDef.Width;
+import org.apache.kafka.common.config.ConfigException;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
 import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
 import java.util.Map;
 
-import io.confluent.common.config.AbstractConfig;
-import io.confluent.common.config.ConfigDef;
-import io.confluent.common.config.ConfigDef.Importance;
-import io.confluent.common.config.ConfigDef.Type;
-import io.confluent.common.config.ConfigException;
-
 public class JdbcSourceConnectorConfig extends AbstractConfig {
 
   public static final String CONNECTION_URL_CONFIG = "connection.url";
   private static final String CONNECTION_URL_DOC = "JDBC connection URL for the database to load.";
+  private static final String CONNECTION_URL_DISPLAY = "Connection Url";
 
   public static final String POLL_INTERVAL_MS_CONFIG = "poll.interval.ms";
   private static final String POLL_INTERVAL_MS_DOC = "Frequency in ms to poll for new data in "
                                                      + "each table.";
   public static final int POLL_INTERVAL_MS_DEFAULT = 5000;
+  private static final String POLL_INTERVAL_MS_DISPLAY = "Poll Interval (ms)";
 
   public static final String BATCH_MAX_ROWS_CONFIG = "batch.max.rows";
   private static final String BATCH_MAX_ROWS_DOC =
       "Maximum number of rows to include in a single batch when polling for new data. This "
       + "setting can be used to limit the amount of data buffered internally in the connector.";
   public static final int BATCH_MAX_ROWS_DEFAULT = 100;
+  private static final String BATCH_MAX_ROWS_DISPLAY = "Batch Max Rows";
```

Comment:
The length of the display name should be close to the length of the typical value for the config.  How about "Max rows per batch" ?